site/en/docs/webstore/monetize-firebase/index.md

```diff
@@ -0,0 +1,544 @@
+
+Begin by creating a directory and the following starter files:
+
+**manifest.json**
+
+Create a file called `manifest.json` and include the following code:
+
+```json
+{
+  "name": "Stripe Firebase Extension",
+  "version": "1.0",
+  "manifest_version": 3,
+  "description": "Stripe Firebase Chrome Extension example",
+  "action": {
+    "default_popup": "popup.html"
+  },
+  "background": {
+    "service_worker": "background.js",
+    "type": "module"
+  },
+  "permissions": ["storage"],
+}
+```
+
+**popup.html**
+
+Create a file called `popup.html` and include the following code:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Firebase Auth Example</title>
+  </head>
+  <body>
+    <h1 id="authState">Checking auth state...</h1>
+    <h2 id="paid">Free account</h2>
+    <input type="email" id="email" placeholder="Email" />
+    <input type="password" id="password" placeholder="Password" />
+    <div style="display: block;">
+      <button id="sign-in">Sign In</button>
+      <button id="sign-out">Sign Out</button>
+    </div>
+    
+    <p>Not registered? <a href="https://payments-with-firebase-example.web.app/" target="_blank">Create an account</a></p>
+    <script src="popup.js" type="module"></script>
+  </body>
+</html>
+```
+
+Replace `https://payments-with-firebase-example.web.app/` with your web app's url.
+
+**popup.js**
+
+Create a file called `popup.js` and include the following code:
+
+```js
+const authState = document.getElementById("auth-state");
+const email = document.getElementById("email");
+const password = document.getElementById("password");
+const payState = document.getElementById("paid")
+const signInButton = document.getElementById("sign-in");
+const signOutButton = document.getElementById("sign-out");
+```
+
+{% include 'partials/extensions/reusing-prod-extension-id.md' %}
+
+## Add Firebase to your extension {: #fb-ext}
+
+Take the following steps to configure Firebase in your extension:
+
+### Get ESM Firebase files {: #fb-esm }
+
+Download the following ESM Firebase files to your project:
+ - [`firebase-app.js`][esm-firebase-app].
+ - [`firebase-auth.js`][esm-firebase-auth].
+ - [`firebase-firestore.js`][esm-firebase-firestore].
+
+{% Aside %}
+
+To download the latest version of these Firebase files, check the [Firebase Javascript SDK release
+notes](https://firebase.google.com/support/release-notes/js) and change the version in the URL to
+the latest version.
+
+{% endAside %}
+
+Update the import statements at the top of `firebase-auth.js` and `firebase-firestore.js` to
+reference the local version of `firebase-app.js`. For example, the following `import` points to a
+local file:
+
+```javascript/6-7
+import {
+  _getProvider,
+  _registerComponent,
+  SDK_VERSION,
+  registerVersion,
+  getApp,
+} from "./firebase-app.js";
```

Comment:
Should we note that the developer should update the path used in this import statement to match where they saved the ESM Firebase files in their project?

Reply:
That seems like a reasonable addition.